sleap/io/dataset.py

```diff
@@ -2618,6 +2618,7 @@ def video_callback(
                         for i, filename in enumerate(filenames):
                             if missing[i]:
                                 filenames[i] = new_paths[i]
+                                missing[i] = False
```

Comment:
Flagging this since it is outside of the scope of the PR (noticed that when loading file through API, found videos are not replaced if USE_DUMM_FOR_MISSING_VIDEOS was set to True) - this line fixes that and only uses dummy for truly missing videos